src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements FigureInterfaces {
```

Comment:
```suggestion
public class Circle extends Figure  {
```
если класс Figure имплементит FigureInterfaces, то все его наследники автоматически его импелментируют

Reply:
Что на счет этого комментария?